ai-platform-unified/notebooks/unofficial/sdk/Vertex_AI_SDK_AutoML_Forecasting_Model_Training_Example.ipynb

```diff
@@ -435,6 +435,84 @@
       "execution_count": null,
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #49.          "templateId": "067f70d2-8cd6-4a4c-a099-292acd1053e8",
</pre><p>How do you ensure that this template won't be deleted?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/399/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
The solution is not 100% satisfying: We created the template, we own the template, hence we'll make sure not to delete it.